zone/raids.cpp

```diff
@@ -1206,8 +1306,13 @@ void Raid::SendMakeGroupLeaderPacketTo(const char *who, Client *to)
 
 void Raid::SendGroupUpdate(Client *to)
 {
-	if(!to)
+	if (!to) {
 		return;
+	}
+
+	if (RuleB(Bots, Enabled) && members[GetPlayerIndex(to)].IsBot) {
```

Comment:
This seems crash prone, we should check that player index is valid and not out of bounds